SPECS/hdf5/hdf5.spec

```diff
@@ -0,0 +1,1054 @@
+%global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
+# No more Java on i686
+%ifarch %{java_arches}
```

Comment:
We don't have `java_arches` defined in our build environment, so we shouldn't use it (just assume Java is there - see my comment below). I'm guessing this is some Fedora-specific macro,